app/boards/shields/m60/m60.keymap

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+/ {
+	keymap0: keymap {
+		compatible = "zmk,keymap";
+
+		default_layer {
+// ------------------------------------------------------------------------------------------
+// | ESC |  1  |  2  |  3  |  4  |  5  |  6  |  7  |  8  |  9  |  0  |  -  |  =  |   BKSP   |
+// | TAB  |  Q  |  W  |  E  |  R  |  T  |  Y  |  U  |  I  |  O  |  P  |  [  |  ]  |    \    |
+// | CAPS  |  A  |  S  |  D  |  F  |  G  |  H  |  J  |  K  |  L  |  ;  |  '  |     ENTER    |
+// |  SHIFT  |  Z  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  |  /  |      SHIFT       |
+// |  CTL  |  WIN  |  ALT  |            SPACE               | ALT | MO(1) |  WIN  | CTRL    |
+// ------------------------------------------------------------------------------------------
+			bindings = <
+	&kp ESC &kp NUM_1 &kp NUM_2 &kp NUM_3 &kp NUM_4 &kp NUM_5 &kp NUM_6 &kp NUM_7 &kp NUM_8 &kp NUM_9 &kp NUM_0 &kp MINUS &kp EQL  &kp BKSP
+	&kp TAB  &kp   Q   &kp   W   &kp   E   &kp   R   &kp   T   &kp   Y   &kp   U   &kp   I   &kp   O   &kp   P   &kp LBKT &kp RBKT &kp BSLH
+	&kp CLCK  &kp   A   &kp   S   &kp   D   &kp   F   &kp   G   &kp   H   &kp   J   &kp   K   &kp   L   &kp  SCLN  &kp  QUOT        &kp RET
+	&kp LSFT   &kp   Z   &kp   X   &kp   C   &kp   V   &kp   B   &kp   N   &kp   M   &kp  CMMA &kp  DOT  &kp FSLH      &kp RSFT
+	&kp LCTL &kp   LGUI   &kp   LALT                     &kp SPC                         &kp  RALT  &mo  1  &kp  RGUI  &kp  RCTL
+			>;
+		};
+
+		fn_layer {
+			bindings = <
+&kp GRAV   &kp F1     &kp F2     &kp F3     &kp F4     &kp F5     &kp F6     &kp F7     &kp F8     &kp F9     &kp F10    &kp F11    &kp F12    &bootloader
+&trans         &bt BT_CLR &none      &none      &none      &none      &none      &none      &none      &none      &none      &none      &none      &reset
+&trans           &bt BT_SEL 0 &bt BT_SEL 1 &bt BT_SEL 2 &bt BT_SEL 3 &bt BT_SEL 4 &kp LARW   &kp DARW   &kp UARW   &kp RARW   &none      &none      &trans
+&trans                &none      &none      &none      &none      &none      &none      &none      &none      &none      &none      &trans
+&trans       &trans       &trans       &trans                                                                     &trans     &trans     &trans     &trans
+			>;
+		};
+
+		unknown_layer {
+			bindings = <
+&none      &none      &none      &none      &none      &none      &none      &none      &none      &none      &none      &none      &none      &kp NUM_1
```

Comment:
Added this layer to use for alignment. I was not able to define a layer with all none keys.
I think it is a bug ?? So as  a workaround, changed a single key to NUM_1

Reply:
Yeah, that seems to be a current bug w/ upstream Zephyr.

Can we just remove this layer altogether? You end up using a bunch of extra space in the final build uf2, I'm not really sure what for?